stage1/app-rm/app-rm.go

```diff
@@ -59,7 +69,31 @@ func main() {
 	}
 
 	enterCmd := stage1common.PrepareEnterCmd(false)
+	switch flagStage {
```

Comment:
Calling this stage0 vs stage1 is a bit confusing - it only refers to the mount namespace (stage0 vs. stage1) - even the "stage 0" cleanup code is removing things created by the mass of code we call "the stage1".

Perhaps a better naming scheme is in order.